Homework_11/Task_1.py

```diff
@@ -0,0 +1,22 @@
+def celsius_to_fahrenheit(celsius: float):
+    return (celsius * 9/5) + 32
+
+def fahrenheit_to_celsius(fahrenheit: float):
+    return (fahrenheit - 32) * 5/9
+
+def convert_temperature(temperature: float, unit: str):
+    try:
+        if unit.upper() == 'C':
+            return celsius_to_fahrenheit(temperature)
+        elif unit.upper() == 'F':
+            return fahrenheit_to_celsius(temperature)
+        else:
+            raise ValueError("Error. Please use 'C' for Celsius or 'F' for Fahrenheit.")
+    except ValueError as e:
+        return str(e)
```

Comment:
This variant
![image](https://github.com/senya2006/Semen-Homework/assets/51053483/ef029c02-f7b5-4ae7-9fc7-c2710a58c3c9)

Reply:
or this variant
![image](https://github.com/senya2006/Semen-Homework/assets/51053483/0b676d3f-5584-4966-a528-68eb224ffa0e)

Reply:
also can be
![image](https://github.com/senya2006/Semen-Homework/assets/51053483/02ae42f7-9338-4a83-a46d-81561e0e5885)

Reply:
First variant will not work because in method "try" we need to use in the end "except" or "finally" so only second variant will work without error...